tools/gcc-bridge/maven-plugin/pom.xml

```diff
@@ -31,6 +31,17 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>org.codehaus.plexus</groupId>
```

Comment:
It looks like this addition is failing the build: 
https://jenkins.bedatadriven.com/job/Renjin/job/renjin-pull-requests/32/console

`plexusCompilerVersion` is not defined.